packages/flutter/test/widgets/navigator_test.dart

```diff
@@ -91,6 +91,31 @@ class OnTapPage extends StatelessWidget {
   }
 }
 
+class SlideInOutPageRoute<T> extends PageRouteBuilder<T> {
+  SlideInOutPageRoute({WidgetBuilder bodyBuilder, RouteSettings settings}) : super(
+    settings: settings,
+    pageBuilder: (BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation) => bodyBuilder(context),
+    transitionsBuilder: (BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation, Widget child) {
+        return SlideTransition(
+          position: Tween<Offset>(
+            begin: const Offset(1.0, 0),
+            end: Offset.zero,
+          ).animate(animation),
+          child: SlideTransition(
+            position: Tween<Offset>(
+              begin: Offset.zero,
+              end: const Offset(-1.0, 0),
+            ).animate(secondaryAnimation),
+            child: child,
+          ),
+        );
+      },
+  );
+
+  @override
```

Comment:
This seems redundant?

Reply:
This is necessary to avoid an analyze error for accessing a protected member.

Reply:
Oh I didn't notice this is protected member.